src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -1210,6 +1173,47 @@ private ObTableBatchOperation buildObTableBatchOperation(List<KeyValue> keyValue
         return batch;
     }
 
+    private com.alipay.oceanbase.rpc.mutation.Mutation buildMutation(KeyValue kv, boolean putToAppend) {
+        KeyValue.Type kvType = KeyValue.Type.codeToType(kv.getType());
+        switch (kvType) {
+            case Put:
+                ObTableOperationType operationType;
+                if (putToAppend) {
+                    operationType = APPEND;
+                } else {
+                    operationType = INSERT_OR_UPDATE;
+                }
+                return com.alipay.oceanbase.rpc.mutation.Mutation.getInstance(operationType, ROW_KEY_COLUMNS,
```

Comment:
Try to import on header

Reply:
There is a Mutation with the same name in org.apache.hadoop.hbase.client，is there any better way to solve this problem?